cpp/src/fil/infer.cu

```diff
@@ -23,6 +23,8 @@
 #include <fil/internal.cuh>
 #include "common.cuh"
 
+#define NOINLINE_FAST_COMPILE __noinline__
```

Comment:
I think `INLINE_CONFIG` would be a better name, given that it can be either `__noinline__` or `__forceinline__`.

Reply:
What do you think about leaving the overall inline/noinline `#define` to a different name, so that `INLINE_CONFIG` means inline/noinline ALL functions (useful for GPU debugging since `-G` tends to still inline `__forceinline__` functions)